test/plug/logger_test.exs

```diff
@@ -74,6 +85,14 @@ defmodule Plug.LoggerTest do
     assert output =~ "Plug.LoggerTest.MyHaltingPlug halted in :halter/2"
   end
 
+  test "logs proper log level to console" do
+    [first_message, second_message] = capture_log_lines fn ->
+      MyDebugLevelPlug.call(conn(:get, "/"), [])
+    end
+    assert Regex.match?(~r/\[debug\] GET \//u, first_message)
```

Comment:
Can you please write it as: `assert first_message =~~ r"\[debug\] GET /"` and similar to the one below? It has a bit less noise. Thank you!